pom.xml

```diff
@@ -162,9 +170,32 @@
       <version>${slf4j.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.mortbay.jetty.alpn</groupId>
+      <artifactId>jetty-alpn-agent</artifactId>
+      <version>${jetty.alpnAgent.version}</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.netty</groupId>
+      <!--<artifactId>netty-tcnative</artifactId>-->
+      <artifactId>netty-tcnative-boringssl-static</artifactId>
+      <classifier>${tcnative.classifier}</classifier>
+      <version>1.1.33.Fork15</version>
+      <scope>test</scope>
+    </dependency>
   </dependencies>
 
   <build>
+
+    <extensions>
+      <extension>
+        <groupId>kr.motd.maven</groupId>
+        <artifactId>os-maven-plugin</artifactId>
+        <version>1.4.1.Final</version>
```

Comment:
we need to ensure that the resulting jar is OS agnostic. (that's more a note for myself)